src/Command/Anonymization/AnonymizeCommand.php

```diff
@@ -173,7 +173,7 @@ protected function interact(InputInterface $input, OutputInterface $output): voi
             return;
         }
 
-        if (null === $this->doBackupAndRestoreInitial) {
+        if (false !== $this->doBackupAndRestoreInitial) {
```

Comment:
Why not simply `if ($this->doBackupAndRestoreInitial)` ? Is `null` still allowed (cf. my comment upper) ?